docs/source/examples/structure_tutorial.rst

```diff
@@ -211,6 +211,7 @@ import phase different kinds will be created::
   >>> asecell = ase.Atoms('Fe2')
```

Comment:
would you mind renaming `asecell` to `atoms` or `ase_atoms`?
that makes it clearer what it represents.

Reply:
maybe `ase_structure` maybe? we're just setting the atom to the abject

Reply:
sure